client/src/lib.rs

```diff
@@ -21,6 +21,7 @@
 
 pub mod att_station;
 pub mod attestation;
+pub mod cli;
```

Comment:
Is it possible not to store parts of CLI code inside the library?

Reply:
Sure, thanks! It shouldn't be a library module. I submitted a fix for this